neuralprophet/forecaster.py

```diff
@@ -1547,14 +1547,19 @@ def set_plotting_backend(self, plotting_backend):
             Specifies plotting backend to use for all plots. Can be configured individually for each plot.
 
             Options
+                * (default) ``plotly-resample``: Use the plotly backend for plotting in resample mode. This mode uses the
+                    plotly-resampler package to accelerate visualizing large data by resampling it. Only supported for
+                    jupyterlab notebooks and vscode notebooks.
                 * ``plotly``: Use the plotly backend for plotting
-                * (default) ``matplotlib``: use matplotlib for plotting
+                * ``matplotlib``: use matplotlib for plotting
         """
-        if plotting_backend in ["plotly", "matplotlib"]:
+        if plotting_backend in ["plotly-auto", "plotly", "matplotlib", "plotly-resampler"]:
             self.plotting_backend = plotting_backend
```

Comment:
Are you intentionally allowing to set "plotly-auto" as a backend?
I would expect you to call `auto_set_plotting_backend` and save the returned backend.